### DISCUSSION TITLE
German localization

### DISCUSSION THREAD

**Discussion by 24Nomad** [Jagusti/fvtt-wfrp4e-gmtoolkit#142] [Q&A] [answered]
Hi,
i like the tool very much i tried to localize it in german. Would be happy if you can include the German localization.
Of course I'm happy to help with further translations or corrections.
Thank you.
24Nomad

[de_json.zip](https://github.com/Jagusti/fvtt-wfrp4e-gmtoolkit/files/9182660/de_json.zip)

**Comment by Jagusti** [comment, marked as answer]
Hey there 24Nomad ... thanks for sharing your enthusiasm for the module, and taking the time to do translations. I'm very happy to accept further localization. I'll add the de.json file to the codebase ...

There are a couple of oddities that it's worth knowing about. 
- Some of the translation strings you've added use double quotes inside double quotes ("), so I've changed the internal quotes to single quotes (') to reflect standard syntax.  
- I've also incorporated some new keys from the latest round of bug fixes. 

Once again, thank you for contributing to the GM Toolkit. Feel free to reach out on Discord (I'm @Jagusti#3610) if you want to discuss anything about the module.
